lib/plugins/aws/deploy/lib/cleanupS3Bucket.js

```diff
@@ -25,7 +25,7 @@ module.exports = {
       .then(response => {
         const stacks = findAndGroupDeployments(response, prefix, service, stage);
         const stacksToKeep = stacks.slice(-stacksToKeepCount || Infinity);
-        const stacksToRemove = _.pullAllWith(stacks, stacksToKeep, _.isEqual);
+        const stacksToRemove = stacks.filter(i => !stacksToKeep.includes(i));
```

Comment:
From what I see logic above is about splitting `stacks` into two slices `stacksToKeep` and `stacksToRemove`, and In that case I think `stacksToRemove` can also be constructed with `slice` same as `stacksToKeep`:

```javascript
stacksToRemove = stacks.slice(0, -stacksToKeepCount || Infinity);
```

I think above notation will be more accurate

Reply:
You are right, I focused replacing lodash functions  #7747 from this issue. So, I could not realize this way.  Thanks